README.md

```diff
@@ -147,9 +147,10 @@ for other operating systems. These packages are not guaranteed to be up-to-date.
     * `eopkg install micro` (Solus).
     * `pacstall -I micro` (Pacstall).
     * See [wiki](https://github.com/zyedidia/micro/wiki/Installing-Micro) for details about CRUX, Termux.
-* Windows: [Chocolatey](https://chocolatey.org) and [Scoop](https://github.com/lukesampson/scoop).
+* Windows: [Chocolatey](https://chocolatey.org), [Scoop](https://github.com/lukesampson/scoop) and [Winget](https://github.com/microsoft/winget-cli).
```

Comment:
Scoop seems to have a website now but I think the URL can be changed in this pull request. A different URL seems to be referenced as the main page of winget in other programs ([git](https://git-scm.com/download/win), [jq](https://jqlang.github.io/jq/download/), [yt-dlp](https://github.com/yt-dlp/yt-dlp/wiki/Installation#winget)). "WinGet" also seems to be written as the name when not in lowercase in official documentation. I do not think there is anything else that has to be changed so that the pull request will be merged.

```suggestion
* Windows: [Chocolatey](https://chocolatey.org), [Scoop](https://scoop.sh/) and [WinGet](https://learn.microsoft.com/en-us/windows/package-manager/winget/).
```